cmd/talosctl/cmd/siderolink/agent.go

```diff
@@ -0,0 +1,61 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// Package siderolink provides the talosctl siderolink-launch command implementation.
+package siderolink
+
+import (
+	"os"
+	"os/signal"
+
+	"github.com/spf13/cobra"
+)
+
+var siderolinkFlags struct {
+	joinToken         string
+	wireguardEndpoint string
+	sinkEndpoint      string
+	apiEndpoint       string
+	logEndpoint       string
+	predefinedPairs   []string
+}
+
+var siderolinkCmd = &cobra.Command{
+	Use:    "siderolink-launch",
```

Comment:
why do we need another folder? can we put it next to other `*-launch` commands, and we can make the whole file `//go:build linux` to have less moving parts

Reply:
Done. Moved to `cmd/talosctl/cmd/mgmt/siderolink_launch_linux.go`